Resources/Prototypes/Traits/species.yml

```
@@ -66,6 +68,33 @@
       traits:
         - Swashbuckler
         - Spearmaster
+        - Mystic
+
+- type: trait
+  id: Mystic
+  category: Mental
+  points: -1
+  functions:
+    - !type:TraitReplaceComponent
+      components:
+      - type: OniDamageModifier
+        modifiers:
+          coefficients:
+            Blunt: 1.1
+            Slash: 1.1
+            Piercing: 1.1
+      - type: PotentiaModifier
+        potentiaMultiplier: 1.5
```

```suggestion
        potentiaMultiplier: 1.15
```
I really wouldn't keep it that high to be honest given that Oni have a lot going for them other than melee damage. Higher stam threshold, superior mass and strength interactions, high damage resistances. This trait lowkey makes Oni pretty comically unbalanced actually, since it outright removes one of their two innate negative traits. 

I would probably suggest instead of nerfing their Oni Damage Modifier, have it be that it removes the OniDamageModifier outright. Since that'll balance it out a lot more if you're removing one of their negatives, you should also remove one of their positives.

---

My thought on this was, they'd still have to take Latent Psionic (altho I should just add latent to this trait) which locks em out of the cybernetics.